mgmt_upgrade_test.py

```diff
@@ -66,8 +66,9 @@ def test_upgrade(self):  # pylint: disable=too-many-locals,too-many-statements
             pre_upgrade_backup_task_files = mgr_cluster.get_backup_files_dict(backup_task_snapshot)
 
         with self.subTest("Creating a backup task and stopping it"):
+            legacy_args = "--force" if manager_tool.client_version.startswith("2.1") else None
```

Comment:
next time please use `distutils.version.LooseVersion` for handling versions
https://android.googlesource.com/toolchain/python/+/0d80e55d38276ed373a55fa4a8f69d62ef5d7408/Python-2.7.3/Lib/distutils/version.py#228